### ISSUE TITLE
I can't seem to get neuralnet trading strategy working with config file

### ISSUE THREAD

**Issue by ampyxx** [askmike/gekko#2764]
**Note: this is the technical bug tracker, please use other platforms for getting support and starting a (non technical) discussion. See the [getting help page](https://gekko.wizb.it/docs/introduction/getting-help.html) for details.**

**I'm submitting a ...**
[ x ] bug report
[ ] question about the decisions made in the repository

**Action taken** (what you did)
Here's my config file:

config.watch = {

  // see https://gekko.wizb.it/docs/introduction/supported_exchanges.html
  exchange: 'binance',
  currency: 'USDT',
  asset: 'BCHABC',

  // You can set your own tickrate (refresh rate).
  // If you don't set it, the defaults are 2 sec for
  // okcoin and 20 sec for all other exchanges.
  // tickrate: 20
}

// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
//                       CONFIGURING TRADING ADVICE
// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

config.tradingAdvisor = {
  enabled: true,
  method: 'neuralnet',
  candleSize: 20,
  historySize: 0,
}

// Neural Network settings:
config.neuralnet = {
	"threshold_buy": 1.0,
	"threshold_sell": -1.0,

	"learning_rate": 1.0,
	"momentum" : 0.5,
	"decay" : 0.10,
	"stoploss_enabled" : false,
	"stoploss_threshold" : 0.85,
	"hodl_threshold" : 1.2,
	"price_buffer_len" : 100,
	"min_predictions" : 1000
}

**Expected result** (what you hoped would happen)

I expected it to trade using the neural network strategy

**Actual result** (unexpected outcome)

It failed to trade completely costing me a good trade. 

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

I'll send you some BCH if you can help me with this bug as it's costing me hundreds of dollars.

**Comment by Subway4711** [comment]
learning rate seems a bit high (i use 0.01) but i have had issues with neuralnet too. Try neuralnetv2 :) https://github.com/xFFFFF/Gekko-Strategies/tree/master/neuralnet_v2

**Comment by ampyxx** [comment]
The weird thing is it seems to be working all of a sudden. Why do you have the learning rate so much lower?

**Comment by Subway4711** [comment]
just because of backtesting. Had the best results with 0.01

**Comment by ampyxx** [comment]
I'm just going to stay with RSI BULL BEAR I think that'll do well
